crates/bevy_utils/src/lib.rs

```diff
@@ -42,8 +42,13 @@ pub use hashbrown;
 #[cfg(feature = "std")]
 pub use parallel_queue::*;
 pub use time::*;
+
+#[cfg(feature = "tracing")]
 pub use tracing;
 
+#[cfg(not(feature = "tracing"))]
+pub use log as tracing;
```

Comment:
Comment here please :) This is a mysterious setup for someone to stumble across.

Reply:
Good point! Included a little comment explaining why this is being done.